config/jobs/kubernetes/sig-node/node-kubelet.yaml

```diff
@@ -264,7 +264,7 @@ periodics:
       - --deployment=node
       - --gcp-project-type=node-e2e-project
       - --gcp-zone=us-west1-b
-      - --node-args=--image-config-file=/workspace/test-infra/jobs/e2e_node/image-config-serial.yaml
+      - --node-args=--image-config-file=/workspace/test-infra/jobs/e2e_node/image-config-serial-swap.yaml
```

Comment:
Any reason to not reuse https://github.com/kubernetes/test-infra/blob/cc714da33d7ba85672aa4c7f58e0b3993155176d/jobs/e2e_node/swap/image-config-swap.yaml ?

Reply:
It looks like I replaced the config on all jobs using it, so there's no point on having two separate ones. Merging them...